libs/wcm/constants/permissions.js

```diff
@@ -1,4 +1,5 @@
 export const SIGNING_METHODS = {
   SIGN_TRANSACTION: { key: 'sign_transaction', title: 'Signature request' },
   SIGN_MESSAGE: { key: 'sign_message', title: 'Sign message' },
+  SIGN_CLAIM_MESSAGE: { key: 'sign_claim_message', title: 'Sign claim message' },
```

Comment:
Maybe this could be more generic since it's just signing raw message. something like `SIGN_MESSAGE_V2` or `SIGN_RAW_MESSAGE`?

Reply:
Done

Reply:
i think you shoulnd't change the original one but update the newly added one? Also, key and title should be updated?

Reply:
Done.